### PR TITLE
docs: add small note about version pinning

### PR DESCRIPTION
**Pull request by david-caro**
Signed-off-by: David Caro david@dcaro.es

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -30,6 +30,16 @@ User's Guide
 This part of the documentation will show you how to get started in using
 inspire-schemas.
 
+**NOTE**: is highly encouraged to pin the version of inspire-schemas that you
```
Comment (kaplun) [inline docs/index.rst]: it is highly...

File: docs/index.rst [hunk 2]
```diff
@@ -30,6 +30,16 @@ User's Guide
...
+use to the major number, for example using the `~=` version matcher in your
+`requires.txt` or your `setup.py` like::
```
Comment (kaplun) [inline docs/index.rst]: `requirements.txt`.

File: docs/index.rst [hunk 3]
```diff
@@ -30,6 +30,16 @@ User's Guide
...
+
+    inspire-schemas~=1.1.0
+
+That will prevent any major update (compatibility breaking) to be pulled
+automatically, and will only download minor and feature updates (1.X.Y).
```
Comment (jacquerie) [inline docs/index.rst]: Not according to PEP 440: https://www.python.org/dev/peps/pep-0440/#compatible-release

Comment (kaplun) [inline docs/index.rst]: Right. It will only upgrade to 1.1.X

**Pull request closed without merging**